integration-tests/src/main/java/org/apache/druid/testing/utils/DruidClusterAdminClient.java

```diff
@@ -185,7 +187,23 @@ private void waitUntilInstanceReady(final String host)
             return response.getStatus().equals(HttpResponseStatus.OK);
           }
           catch (Throwable e) {
-            LOG.error(e, "");
+            //
+            // supress logging for some specific exceptions to reduce excessive stack trace messages when waiting druid nodes to start up
+            //
+            if (e.getCause() instanceof ChannelException) {
+              Throwable channelException = e.getCause();
+
+              if (channelException.getCause() instanceof ClosedChannelException) {
+                LOG.error("Channel Closed");
```

Comment:
I think we should log the message just incase there is some interesting information in there. `getMessage()` should only be 1 line

```suggestion
                LOG.error("Channel Closed " + e.getMessage());
```

Reply:
`ClosedChannelException` class has no constructor to accept message parameter, so `getMessage` call here outputs `null`
```
public class ClosedChannelException
    extends java.io.IOException
{

    private static final long serialVersionUID = 882777185433553857L;

    /**
     * Constructs an instance of this class.
     */
    public ClosedChannelException() { }

}
```